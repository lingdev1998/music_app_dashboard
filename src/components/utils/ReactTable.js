import React from 'react';
import styled from 'styled-components';
import ReactTable from "react-table";
import 'react-table/react-table.css';

const StyledReactTable = styled(ReactTable)`
    .ReactTable{
        background-color: ${props => props.theme.backgroundColor};
        color: ${props => props.theme.color};       
    }    
`
const AppReactTable = (props) => <StyledReactTable {...props} />
export default AppReactTable;