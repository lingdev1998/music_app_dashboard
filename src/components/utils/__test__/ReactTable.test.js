import React from 'react';
import { shallow } from "enzyme";
import ReactTable from '../ReactTable'
import toJson from "enzyme-to-json";
describe('<ReactTable/>', () => {
    const wrapper = shallow(<ReactTable />)
    it('should matches snapshot', () => {
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})
