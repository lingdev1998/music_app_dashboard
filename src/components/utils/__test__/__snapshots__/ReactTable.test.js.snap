// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReactTable/> should matches snapshot 1`] = `<Styled(ReactTable) />`;
